openmeetings-service/src/main/java/org/apache/openmeetings/service/mail/template/InvitationTemplate.html

```diff
@@ -16,20 +16,37 @@
   KIND, either express or implied.  See the License for the
   specific language governing permissions and limitations
   under the License.
-
 -->
 <!DOCTYPE html>
 <html xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-9.xsd">
+
+<head>
```

Comment:
This `<head>...</head>` should be dropped :)

Reply:
Heloo. Sorry, my bad